notebooks/iceflow/3_dataviz.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center>\n",
+    "<img src='./img/nsidc_logo.png'/>\n",
+    "\n",
+    "# **IceFlow**\n",
+    "### Point Cloud Data Access\n",
+    "</center>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## Visualizing large datasets\n",
+    "Valkyrie and Icesat 2 data are by nature big data sets that require some special considerations when working with it. The main constraint if you don't have a super computer is memory. The average granule size is in the 10s of MB for IceSat 2 and could be Gigabytes in Valkyrie depending on the order/subsetting. \n",
+    "\n",
+    "This is when libraries like Dask, Vaex and others come into play. This notebook will show you how to use some basic plotting techniques using matpplotlib and geopandas + Vaex to work effectively with lidar data from Valkyrie and ATL data from IceSat 2.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "import glob\n",
+    "import geopandas\n",
+    "import pandas as pd\n",
+    "import h5py\n",
+    "import vaex\n",
+    "import dask.dataframe as dd\n",
+    "import dask.array as da\n",
+    "import numpy as np\n",
+    "from iceflow.processing import IceFlowProcessing as ifp\n",
+    "\n",
+    "# filepath = 'data/atm1b_data_2020-07-10T15-32.hdf5'\n",
+    "# df_k = ifp.get_common_dictionary('ATM')\n",
+    "\n",
+    "filepath = 'data/twaties-test-GLAH06-2000-2010.h5'\n",
+    "df_key = ifp.get_common_dictionary('GLASS')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading the data with H5PY"
```

Comment:
I received this error from the LVIS output from iceflow: 

`AttributeError: 'NoneType' object has no attribute 'describe'`